unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendsHUD/Scripts/FriendsHUDController.cs

```diff
@@ -402,6 +405,7 @@ private void DisplayMoreFriends()
         if (!friendsController.IsInitialized) return;
         ShowOrHideMoreFriendsToLoadHint();
         friendsController.GetFriends(LOAD_FRIENDS_ON_DEMAND_COUNT, View.FriendCount);
+        isFriendsLoadedForFirstTime = true;
```

Comment:
Wouldn't be cleaner to put the `isFriendsLoadedForFirstTime` condition at the beginning of the `DisplayMoreFriends()` function instead of doing it before all the calls to `DisplayMoreFriends()`? Something like this:

```
DisplayMoreFriends()
{
    if (isFriendsLoadedForFirstTime) return

    ...

    isFriendsLoadedForFirstTime = true;
}
```

Reply:
not all the flows aligns with this condition.. for example when you are loading more friends while scrolling down